test/activities/content/ContentWebLinkEntity.test.js

```diff
@@ -72,7 +75,18 @@ describe('ContentWebLinkEntity', () => {
 			const webLinkData = {
 				title: 'Test Web Link Title',
 				url: 'https://phoenix-is-the-best.com',
-				isExternalResource: false
+				isExternalResource: false,
+				descriptionRichText: '<p>description text</p>'
+			};
+			expect(contentWebLinkEntity.equals(webLinkData)).to.equal(false);
+		});
+
+		it('Equality should return false when description rich text is different', () => {
+			const webLinkData = {
+				title: 'Test Web Link Title',
+				url: 'https://phoenix-is-the-very-best.com',
```

Comment:
The url got changed as too. It should have been 
`https://phoenix-is-the-best.com`

Reply:
Done